apps/portals/elportal/src/config/synapseConfigs/handleUpsetPlotClick.tsx

```diff
@@ -1,19 +1,29 @@
-import { Query, FacetColumnRequest } from '@sage-bionetworks/synapse-types'
-import { cohortBuilderFilesSql } from '../resources'
+import {
+  Query,
+  ColumnMultiValueFunctionQueryFilter,
+  ColumnMultiValueFunction,
+  COLUMN_MULTI_VALUE_FUNCTION_QUERY_FILTER_CONCRETE_TYPE_VALUE,
+} from '@sage-bionetworks/synapse-types'
+import { cohortBuilderSql } from '../resources'
 
 export const handleUpsetPlotClick = (selection: any) => {
```

Comment:
Is it possible to add the type from upset.js here?
```suggestion
export const handleUpsetPlotClick: UpSetSelectionProps['onClick'] = (selection) => {
```